### PR TITLE
OADP 1.3 DataMover restore hook issue

### PR DESCRIPTION
**Pull request by RichardHoch**
OADP 1.3; OCP 4.11+

Resolves https://issues.redhat.com/browse/OADP-3059 by writing a short note to describe the problem. 

Preview: https://67817--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/data-mover-intro [end of introduction] 

OADP QA approved.

### CODE REVIEW

**Comment by amastbau** [comment]
neeed to check about 1.2
i think it has the same issue,
but i could be wrong

**Comment by amastbau** [comment]
LGTM

**Comment by RichardHoch** [comment]
@anarnold97   I have written the note as a snippet so we can move it easily later, if needed. For now, I have added the snippet to the end of the OADP Data Mover Introduction. If you have a different suggestion for its location, please let me know. I'm asking for a peer review in parallel to save time.

File: snippets/snip-post-mig-hook [hunk 1]
```diff
@@ -0,0 +1,10 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+Post-migration hooks are not likely to work well with the OADP 1.3 Data Mover.
+
+The OADP 1.1 and OADP 1.2 Data Movers use synchronous processes to back up and restore application data. Because the processes are synchronous, users can be sure that any post-restore hooks start only after the persistent volumes (PVs) of the related pods are released by the persistent volume claim (PVC) of the Data Mover.
+
+However, the OADP 1.3 Data Mover uses an asynchronous process. As a result of this difference in sequencing, a post-restore hook might be called before the related PVs were released by the PVC of the Data Mover. If this happens, the pod remains in `Pending` status and cannot run the hook. The hook attempt might time out before the pod is released, leading to a `PartiallyFailed` restore operation.
```
Comment (mburke5678) [inline snippets/snip-post-mig-hook:9]: Any work-around for this?

**Comment by mburke5678** [comment]
@RichardHoch One question. Otherwise LGTM

File: snippets/snip-post-mig-hook (hunk 1 above)
Comment (anarnold97) [reply snippets/snip-post-mig-hook:9]: @amastbau && @weshayutin - please can we check if there is a workaround? i looked at the slack channel but could not see anything
Thanks

Comment (sseago) [reply snippets/snip-post-mig-hook:9]: @weshayutin Right, there's not really a workaround here. I think @shubham-pampattiwar is assigned the actual jira for fixing this? Or maybe I'm misremembering.

Comment (anarnold97) [reply snippets/snip-post-mig-hook:9]: @weshayutin && @sseago - grand so. Thanks very much for the confirmation! caps are doffed 

@shubham-pampattiwar - please can you give us a shout so we can update thanks

**Comment by RichardHoch** [comment]
@mburke5678 Unfortunately, there is no workaround. This should be merged as is with OADP 1.3's GA, which is scheduled for today. Thanks!

**Comment by RichardHoch** [comment]
Please merge today; GA was yesterday.

**Comment by ousleyp** [comment]
LGTM

**Pull request merged**